apps/trading/client-pages/markets/markets-sidebar.tsx

```diff
@@ -11,12 +11,7 @@ import {
 import { useGetCurrentRouteId } from '../../lib/hooks/use-get-current-route-id';
 
 export const MarketsSidebar = () => {
-  const { marketId } = useParams();
   const currentRouteId = useGetCurrentRouteId();
-  const { data } = useMarket(marketId);
```

Comment:
useMarket, useParams can be removed from imports

Reply:
done!